proteus/mprans/SpatialTools.py

```diff
@@ -83,7 +84,7 @@ def _attachAuxiliaryVariable(self, key):
         """
         if key not in self.auxiliaryVariables:
             if key == 'RigidBody':
-                self.auxiliaryVariables[key] = True
+                self.auxiliaryVariables[key] = bd.RigidBody(self)
```

Comment:
I think dt_init and cfl_target need to stay out of Shapes. The RigidBody objects sometimes need to know those, but they don't need to be modifying them. Those are numerical method parameters for the TimIntegration/SplitOperator/StepControl objects.